### DISCUSSION TITLE
Hexahedron Cube as element

### DISCUSSION THREAD

**Discussion by glaand** [inducer/meshpy#85] [Q&A]
I have built a solver for my undergraduate problem and i am battling with the meshing... but that's not something new to the FEM world :P 

At the moment i use simple cube elements and i would like to automate the meshing part of my code and i found your package but you only use tetrahedron elements. I mean sometimes cubes perform better than tetrahedron and have less tendencies for hour glass problems, maybe the implementation of the cube could be something useful for the FEM community.

**Comment by inducer** [comment]
The mesh generators in meshpy are called "Triangle" and "TetGen"... ("tet" for tetrahedra), so I would say meshpy is unlikely to pick up this functionality. Gmsh can generate hex meshes, at least to an extent.
